### ISSUE TITLE
Fix a11y issues in Documentation/React + HTML /Table

### ISSUE THREAD

**Issue by christiemolloy** [patternfly/patternfly-org#1319]
https://www.patternfly.org/v4/documentation/react/components/table/
https://www.patternfly.org/v4/documentation/core/components/table

There is 1 type of a11y issue on this page(the rest are covered in separate issues):
1. scope attribute should be used correctly
2. Checkbox inputs with the same name attribute value must be part of a group
3. scope attribute should be used correctly

<img width="479" alt="Screen Shot 2019-07-17 at 10 30 25 AM" src="https://user-images.githubusercontent.com/20118816/61384110-ec478800-a87d-11e9-98f4-a6de28d8be85.png">

Steps to reproduce the a11y errors:

1. Right click on the page and choose inspect
2. In the developer tools, choose "axe" from the top menu
3. Run the test

**Comment by jessiehuff** [comment]
This issue looks to be solved. 🙂
